content/docs/capabilities/ppl.mdx

```diff
@@ -168,6 +168,9 @@ A certificate matcher can be used to allow or deny specific TLS certificates, ba
 | Key Name | Value Type | Description |
```

Comment:
In the paragraph above this ("A certificate matcher can be used to..."), you mention that a cert matcher can be used to allow or deny certs based on the certificate fingerprint or SPKI hash, but you do not mention the new SAN options in the table below. Was that intentionally omitted or should that be included in the criteria?

Reply:
Good catch! I think the table is descriptive enough on its own, so maybe we can simplify the introductory paragraph like this?

Reply:
Yes, that update makes sense to me.